### PR TITLE
Replace default tuned CR on operator restart.

### PR DESCRIPTION
**Pull request by jmencak**
This commit:
  - replaces default tuned CR on operator restart (e.g. upgrade)
  - makes sure bindata is regenerated out of modified assets
    in case an old version of bindata is already present
  - removes extraneous "for" from logging messages

In the presence of custom CRs not to replace the default CR shipped with the
operator (especially on operator upgrade) is a bug and is addressed by this
commit.  Users should create their own custom CRs instead of modifying the
default one.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,14 +23,14 @@ build: $(BINDATA)
 	$(GO_BUILD_RECIPE)
 
 # Using "-modtime 1" to make generate target deterministic. It sets all file time stamps to unix timestamp 1
-generate $(BINDATA): $(GOBINDATA_BIN)
+$(BINDATA): $(GOBINDATA_BIN) $(ASSETS)
```
Comment (sjug) [inline Makefile]: So the name of this build target is `pkg/manifests/bindata.go`?

Comment (jmencak) [reply Makefile]: Yes.  This will trigger `pkg/manifests/bindata.go` to be re-generated when assets change.

Comment (sjug) [reply Makefile]: ack, just an odd name is all

**Pull request merged**